.travis.yml

```diff
@@ -20,9 +26,20 @@ cache:
 env:
   - MORPH_API_KEY=FakeApiKey # set a fake morph api key so the system checks pass
 install:
-  - pip install pip==8.1.2
+  - sudo unlink /usr/bin/g++ && sudo ln -s /usr/bin/g++-5 /usr/bin/g++
+  - pip install --upgrade pip
   - pip install tox coveralls tox-travis
   - npm install
+  - wget https://github.com/apiaryio/drafter/releases/download/v3.2.7/drafter-v3.2.7.tar.gz
```

Comment:
It's annoying there isn't a `.deb` for this, as compiling for each build will take up a lot of time. 

Is there a way to cache the build? Maybe adding `~/drafter-v3.2.7/build` to the cache dir in travis and checking for `build/out/Release/lib.target/libdrafter.so` before running `configure` and `make`?

Alternatively, how hard is it to make a `.deb` and host it on S3? Or just get the project to manage one after a PR from us?

Reply:
Yes - it is a bit of a pain. They do produce [an `aur` package for Arch Linux ](https://aur.archlinux.org/packages/drafter/) but no `deb`. I suspect that packaging for Brew and Arch reflects the preferences of the developers who maintain it. I have had a cursory look at docs for debian packaging at one point.. based on that, I think build caching is probs the first thing I'll look at ;)